### PR TITLE
Add javadocs for error classes

### PR DESCRIPTION
**Pull request by theodorekwok**
links to #156

### CODE REVIEW

File: src/main/java/seedu/mtracker/error/fileerror/InvalidInstrumentInFileError.java [hunk 1]
```diff
@@ -2,7 +2,16 @@
 
 import seedu.mtracker.error.ErrorMessage;
 
+/**
+ * The custom exception class that is thrown when the storage file instrument type provided is not recognised.
```
Comment (williamlamjy) [inline src/main/java/seedu/mtracker/error/fileerror/InvalidInstrumentInFileError.java:6]: not valid*

File: src/main/java/seedu/mtracker/error/fileerror/InvalidInstrumentInFileError.java [hunk 2]
```diff
@@ -2,7 +2,16 @@
...
+ */
 public class InvalidInstrumentInFileError extends Exception {
+
+    /**
+     * Returns the error message to the user stating that storage file instrument type is not recognised.
```
Comment (williamlamjy) [inline src/main/java/seedu/mtracker/error/fileerror/InvalidInstrumentInFileError.java:11]: not valid*

File: src/main/java/seedu/mtracker/error/fileerror/InvalidRemarksInFileError.java [hunk 3]
```diff
@@ -2,7 +2,16 @@
 
 import seedu.mtracker.error.ErrorMessage;
 
+/**
+ * The custom exception class that is thrown when storage file instrument remarks is saved wrongly.
```
Comment (williamlamjy) [inline src/main/java/seedu/mtracker/error/fileerror/InvalidRemarksInFileError.java:6]: saved in the wrong format*

File: src/main/java/seedu/mtracker/error/fileerror/InvalidRemarksInFileError.java [hunk 4]
```diff
@@ -2,7 +2,16 @@
...
+ */
 public class InvalidRemarksInFileError extends Exception {
+
+    /**
+     * Returns the error message to the user stating that remarks in storage file is saved wrongly.
```
Comment (williamlamjy) [inline src/main/java/seedu/mtracker/error/fileerror/InvalidRemarksInFileError.java:11]: saved in the wrong format*

File: src/main/java/seedu/mtracker/error/fileerror/InvalidSentimentSavedInFileError.java [hunk 5]
```diff
@@ -2,7 +2,16 @@
 
 import seedu.mtracker.error.ErrorMessage;
 
+/**
+ * The custom exception class that is thrown when storage file instrument sentiment is saved wrongly.
```
Comment (williamlamjy) [inline src/main/java/seedu/mtracker/error/fileerror/InvalidSentimentSavedInFileError.java:6]: make everything "is invalid"?

**Review by williamlamjy — COMMENTED** [review]
LGTM just some minor standardisation for phrasing but tbh not v impt

File: src/main/java/seedu/mtracker/error/ErrorMessage.java [hunk 6]
```diff
@@ -1,5 +1,8 @@
 package seedu.mtracker.error;
 
+/**
+ * An ErrorMessage class that displays all the user facing errors.
```
Comment (kum-wh) [inline src/main/java/seedu/mtracker/error/ErrorMessage.java:4]: This one sounds bit confusing.

**Review by kum-wh — COMMENTED** [review]
Not much except the Errormessage class comment sound a bit confusing to me.

**Comment by williamlamjy** [comment]
LGTM can merge after updating

**Comment by williamlamjy** [comment]
LGTM

**Pull request merged**